### ISSUE TITLE
Cannot compile unknown node `login`

### ISSUE THREAD

**Issue by albinekb** [syntax-tree/hast-util-to-html#44]
### Initial checklist

- [X] I read the [support docs](https://github.com/syntax-tree/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/syntax-tree/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/syntax-tree/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Asyntax-tree&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

hast-util-to-html@9.0.3

### Steps to reproduce

```js
const processor = unified()
    .use(rehypeParse, { fragment: true, emitParseErrors: true })
    .use(rehypeFormat, { blanks: [], indent: 0 })
    .use(rehypeStringify, { allowParseErrors: true})
    
const vfile = await processor.process('<button type="login">Click me</button>')

const text = vfile.toString()
```

```
Cannot compile unknown node `login`
    at unknown (node_modules/.pnpm/hast-util-to-html@9.0.3/node_modules/hast-util-to-html/lib/handle/index.js:46:9)
    at one (node_modules/.pnpm/zwitch@2.0.4/node_modules/zwitch/index.js:108:17)
    at Object.one (node_modules/.pnpm/hast-util-to-html@9.0.3/node_modules/hast-util-to-html/lib/index.js:234:10)
    at Object.all (node_modules/.pnpm/hast-util-to-html@9.0.3/node_modules/hast-util-to-html/lib/index.js:253:27)
    at element (node_modules/.pnpm/hast-util-to-html@9.0.3/node_modules/hast-util-to-html/lib/handle/element.js:78:25)
    at one (node_modules/.pnpm/zwitch@2.0.4/node_modules/zwitch/index.js:108:17)
    at Object.one (node_modules/.pnpm/hast-util-to-html@9.0.3/node_modules/hast-util-to-html/lib/index.js:234:10)
    at Object.all (node_modules/.pnpm/hast-util-to-html@9.0.3/node_modules/hast-util-to-html/lib/index.js:253:27)
    at element (node_modules/.pnpm/hast-util-to-html@9.0.3/node_modules/hast-util-to-html/lib/handle/element.js:78:25)
    at one (node_modules/.pnpm/zwitch@2.0.4/node_modules/zwitch/index.js:108:17)
```

### Expected behavior

No error is thrown or I can allow undefined values on attributes when stringifying

### Actual behavior

Crashes when `type` is bogus value

### Affected runtime and version

node@20.17.0

### Affected package manager and version

pnpm@9.4.0

**Comment by wooorm** [comment]
Hi!

- `login` is not a valid value for `type` on `button` — do not write incorrect HTML.
- when you post code, remove everything that is not needed. Remove all the fields of all the options you do not needed, remove all the plugins you do not need, please, thanks.
- can you please post version numbers of all the things? and make sure they are up to date?

**Comment by albinekb** [comment]
Sorry, this is an issue with https://github.com/syntax-tree/hast-util-from-html
I will open the issue over there instead, repro here: https://github.com/albinekb/hast-unknown-node-repro/blob/main/index.ts

**Comment by wooorm** [comment]
Released in [`hast-util-from-parse5@8.0.2`](https://github.com/syntax-tree/hast-util-from-parse5/releases/tag/8.0.2), thank you for working on this!
